target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq8072-rt5010w-d187.dts

```diff
@@ -0,0 +1,250 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/* Copyright (c) 2022, Robert Marko <robimarko@gmail.com> */
```

Comment:
Are you sure this DTS belongs to robimarko

Reply:
fixed